README.md

```diff
@@ -1,6 +1,14 @@
 # Dagster Cloud Serverless Deployment Quickstart
 
-## Create a new project from this template
+Welcome to your Dagster Cloud sample code repo. Here, you can find the code that's being deployed to your Dagster Cloud instance.
+
+Pushing to production will automatically kick off a [workflow](./.github/workflows/deploy.yml) which will redeploy your code to your `prod` deployment.
```

Comment:
we can add a link to https://docs.dagster.io/dagster-cloud/deployment/serverless for more details